Resources/doc/dynamic.md

```diff
@@ -132,6 +132,7 @@ sulu_form:
     dynamic_lists:
         sulu_page.page_edit_form: # view key e.g. "sulu_page.page_edit_form" for sulu pages or "sulu_article.edit_form" for sulu articles.
             form: # unique key mostly the same as the template key or a combination between template and property key.
```

Comment:
as reading this command I think we should just use the `key` as postfix for the viewbuilder, so uniqueness is defined by the yaml key.